dnf/cli/aliases.py

```diff
@@ -178,6 +178,11 @@ def subresolve(args):
                     stack.pop()
                 except IndexError:
                     pass
+
+                # strip the '\\' if it exists
```

Comment:
Can you write one backslash only, instead of two? In this comment and also in the commit message. I think that with double backslash it's a bit confusing, because the escaping is not necessary to escape it in comments and also because you are using single backslash in the commit message, too.